packages/sqls/package.yaml

```diff
@@ -10,7 +10,7 @@ categories:
   - LSP
 
 source:
-  id: pkg:golang/github.com/lighttiger2505/sqls@v0.2.22
+  id: pkg:golang/github.com/lighttiger2505/sqls@v0.2.28
```

Comment:
This repo was also recently transferred to an org: https://github.com/sqls-server/sqls